assets/src/edit-story/app/prepublish/constants.js

```diff
@@ -14,8 +14,332 @@
+        __(
+          'Should be at least %1$s and maintain a %2$s aspect ratio.',
+          'web-stories'
+        ),
+        `${PUBLISHER_LOGO_DIMENSION}x${PUBLISHER_LOGO_DIMENSION}px`,
+        `${PUBLISHER_LOGO_RATIO}x${PUBLISHER_LOGO_RATIO}px`
+      ),
+    },
+    LINK_ATTACHMENT_CONFLICT: {
+      MAIN_TEXT: __('Link conflict with page attachment', 'web-stories'),
+      HELPER_TEXT: __(
+        'Links in the bottom of a page with a Page Attachment are disabled.',
+        'web-stories'
+      ),
+    },
+    MISSING_VIDEO_POSTER: {
+      MAIN_TEXT: __('Video missing poster image', 'web-stories'),
+      HELPER_TEXT: __(
+        'Add a poster image for every video. Posters are displayed while a video loads for a better user experience.',
+        'web-stories'
+      ),
+    },
+  },
+  ACCESSIBILITY: {
+    LOW_CONTRAST: {
+      MAIN_TEXT: __(
+        'Low contrast between font and background color',
+        'web-stories'
+      ),
+      HELPER_TEXT: __(
+        'A high contrast makes the words easy to see.',
+        'web-stories'
+      ),
+    },
+    FONT_TOO_SMALL: {
+      MAIN_TEXT: __('Font size too small', 'web-stories'),
+      HELPER_TEXT: sprintf(
+        /* translators: 1: minimum font size. */
+        __('Text should be size %d or higher.', 'web-stories'),
+        MIN_FONT_SIZE
+      ),
+    },
+    LOW_IMAGE_RESOLUTION: {
+      MAIN_TEXT: __('Low image resolution', 'web-stories'),
+      HELPER_TEXT: sprintf(
+        /* translators: 1: minimum image size width x minimum image size height. */
+        __(
+          'Use %s for full screen images. Use similar pixel density for cropped images.',
+          'web-stories'
+        ),
+        `${IMAGE_SIZE_WIDTH}x${IMAGE_SIZE_HEIGHT}px`
+      ),
+    },
+    MISSING_CAPTIONS: {
+      MAIN_TEXT: __('Video missing captions', 'web-stories'),
+      HELPER_TEXT: __(
+        'Add captions to your video. Captions help keep your audience engaged, even when they can’t listen to the audio.',
+        'web-stories'
+      ),
+    },
+    MISSING_VIDEO_TITLE: {
+      MAIN_TEXT: __('Video missing description', 'web-stories'),
+      HELPER_TEXT: __(
+        'Add a video description. Descriptions help with indexability and accessibility (for videos without captions). Include any burned-in text inside of the video.',
+        'web-stories'
+      ),
+    },
+    TOO_MANY_LINKS: {
+      MAIN_TEXT: __('Too many links on a page', 'web-stories'),
+      HELPER_TEXT: __(
+        'Avoid having more than one link per page.',
+        'web-stories'
+      ),
+    },
+    LINK_REGION_TOO_SMALL: {
+      MAIN_TEXT: __('Link tappable region too small', 'web-stories'),
+      HELPER_TEXT: __(
+        "Make the linked element large enough so it's easy for a user to tap it.",
+        'web-stories'
+      ),
+    },
+    MISSING_IMAGE_ALT_TEXT: {
+      MAIN_TEXT: __('Image missing alt text', 'web-stories'),
+      HELPER_TEXT: sprintf(
+        /* translators: %s: alt. */
```

Comment:
mostly for my own knowledge, is "alt" not something that we'd want translated as part of the string?

Reply:
Pascal's feedback [here](https://github.com/google/web-stories-wp/pull/5347#discussion_r526766881) :)

Reply:
cool!